.nvmrc

```diff
@@ -1 +1 @@
-v18.17.1
```

Comment:
This the sort of change I wouldn't expect in this PR. And instead in a PR that was about upgrading node.

But, I guess this is just some clean up after that fact?